<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QMainWindow" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>日记编辑</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFontComboBox" name="fontComboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>华文楷体</string>
         </property>
         <property name="currentIndex">
          <number>234</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="res.qrc">:/images/IMG_6.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="saveButton">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/images/IMG_4.jpg</normaloff>:/images/IMG_4.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionBack"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionBack">
   <property name="text">
    <string>返回上一级</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
